modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/AbstractSnapshotSelfTest.java

```
@@ -177,7 +178,8 @@ public static Collection<Boolean> encryptionParams() {
                 .setPageSize(DFLT_PAGE_SIZE))
             .setClusterStateOnStart(INACTIVE)
             .setIncludeEventTypes(EVTS_CLUSTER_SNAPSHOT)
-            .setDiscoverySpi(discoSpi);
+            .setDiscoverySpi(discoSpi)
+            .setFailureHandler(new StopNodeFailureHandler());
```

Should we raise this failure handler where needed only?